SPECS/hdf5/hdf5.spec

```diff
@@ -0,0 +1,971 @@
+%global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
+%define version_main %(echo %{version} | cut -d. -f-2)
+%global so_version 200
+%global with_mpich 0
+%global with_openmpi 0
+%if %{with_mpich}
+%global mpi_list mpich
+%endif
+%if %{with_openmpi}
+%global mpi_list %{?mpi_list} openmpi
+%endif
+Summary:        A general purpose library and file format for storing scientific data
+Name:           hdf5
+Version:        1.12.1
+Release:        11%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://portal.hdfgroup.org/display/HDF5/HDF5
+Source0:        https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%{version_main}/hdf5-%{version}/src/hdf5-%{version}.tar.bz2
+Source1:        h5comp
+Patch0:         hdf5-LD_LIBRARY_PATH.patch
+# Fix fortran build with gcc 12
+# https://github.com/HDFGroup/hdf5/pull/1412
+Patch1:         hdf5-gfortran12.patch
+Patch3:         hdf5-build.patch
+# Remove Fedora build flags from h5cc/h5c++/h5fc
+# https://bugzilla.redhat.com/show_bug.cgi?id=1794625
+Patch5:         hdf5-wrappers.patch
+# For patches/rpath
+BuildRequires:  automake
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  gcc-gfortran
+BuildRequires:  hostname
+BuildRequires:  krb5-devel
+BuildRequires:  libaec-devel
+BuildRequires:  libtool
+# Needed for mpi tests
+BuildRequires:  openssh-clients
+BuildRequires:  openssl-devel
+BuildRequires:  time
+BuildRequires:  zlib-devel
+BuildRequires: krb5-devel
+BuildRequires: openssl-devel
+BuildRequires: time
+BuildRequires: zlib-devel
+BuildRequires: hostname
+# For patches/rpath
+BuildRequires: automake
+BuildRequires: libtool
+# Needed for mpi tests
+BuildRequires: gcc
+BuildRequires: gcc-c++
+BuildRequires: libaec-devel
+BuildRequires: openssh-clients
```

Comment:
Please lint the spec - it looks like we have some duplicates and inconsistent spacing.

Reply:
spec linting was crashing when I used the linter with this file. let me try again with all the changes I applied. Except for this hdf5 spec, I lint'd all other spec files.

Reply:
@PawelWMS linting works now. I should have saved what error I was getting yesterday.

Reply:
No worries, we can take a look, if we hit it again.